docs/source/datasets.rst

```diff
@@ -25,9 +25,8 @@ click the link provided in the "Module" column.
 +-------------------+---------------------+---------------------+---------------------------+--------+
 | Module            | Name                | Downloadable?       | Annotation Types          | Tracks |
 +===================+=====================+=====================+===========================+========+
-| beatles_          | | The Beatles       | - audio: ❌         | - :ref:`beats`            | 180    |
-|                   | | Dataset           | - annotations: ✅   | - :ref:`sections`         |        |
-|                   |                     |                     | - :ref:`key`              |        |
+| ab_genre_         | AcousticBrainz      | - audio: ✅         | - :ref:`genre`            | >4mill |
```

Comment:
is this the name of the module in mirdata or acousticbrainz_genre? was it shortened because the table would overflow?

Reply:
Mmmm same concern, it would be good to keep consistency naming the loader. Is it much longer than others? e.g. 
`medleydb_solos_db`
`acousticbrainz_genre`
`acousticbrainz_gen`

Reply:
yes it is bigger.... but well I am going to keep consistency... and extend the table spaces... thanks!